Documentation/Modularization-concept.md

```diff
@@ -1,154 +1,103 @@
+For each module that contains at least one IC or PV BB, the user can select one (or none) IC and/or PV BB.
 
-If a model configuration contains a PK-Sim module, the user must select an Individual (pre-select the first one?) and Expression Profiles (optionally). Individual and Expression Profiles can be selected without a PK-Sim module. In the hierarchy of the selected modules, the Individuals-BB always comes before the extension modules (i.e., the parameters are applied to the standard PK-Sim module). When an extension module is selected, the user must additionally select an IC and PV BB of that module.
+## Combination rules
 
-![Structure of a model configuration](Figures/ModelConfiguration_Scheme.png)
+During simulation creation, the modules are combined to a common model structure. Entities are combined (extended or overwritten) by their absoulte path (for containers in the spatial structure, parameters, and molecule values) or by their names (for neighborhoods, transports, molecules, etc). The result is a simulation which represents the combination of the selected modules.
 
-The model configuration will potentially contain IC values that are not defined in the modules (e.g., a combination of a module with new molecules with a module with a new organ). These values are populated with default values (initial condition e.g. 0). The newly created BBs can be exported and re-imported as new modules, resulting in modules containg the intial conditions for these non-standard combinations.
+### Spatial structure
 
-Model configurations can be **configured** and **cloned**.
+For combining the spatial structures BB, two modes exist. A module can *extend* or *overwrite* the structure created in the previous steps.
 
-## Simulation
-Simulations are created from model configurations. A single model configuration can have multiple models. All these models will have the same structure, but may differ in their parameters and/or initial conditions. During creation of a model from the model configuration, the user can select a different Individual and/or Expression Profile.
+### Merge behavior "Overwrite"
+When combining modules "A" and "B" (with the hierarchy "A" -> "B"), all containers from module "B" will overwrite the containers with the same path in module "A". I.e., if module "A" has a container "Organism|Container 1" with two sub-containers "Container 2" and "Container 3" (absolute paths: "Organism|Container 1|Container 2" and "Organism|Container 1|Container 2"), and module "B" has a container "Organism|Container 1" without any sub-containers, the final model will contain "Organism|Container 1" without the sub-containers "Container 2" and "Container 3". "Organism|Container 1" will only have parameters that are defined in module "B", but not in module "A".
```

Comment:
"A" -> "B" looks for me like "A" overwrites "B" - but in the description it is the other way around. 
I would change the arrow direction: "A" <- "B"

Reply:
Adding a picture which demonstrates the behavior of each use case would be of great value

Reply:
https://github.com/Open-Systems-Pharmacology/OSMOSES/issues/66